### DISCUSSION TITLE
Opening the repository with the Dev Container is stuck at the conda installation

### DISCUSSION THREAD

**Discussion by WouterOffringa** [tudelft/AE4353-Y24#8] [Q&A] [answered]
I have followed all the steps and am at the final step of the Usage part of the readme. When opening the AE4353-Y24 folder and looking at the progress log of the Dev Container, this seems to take endlessly. I left my computer running for 2+ hours and it still did not open the folder. I tried a second time after restarting my laptop and it still does not continue beyond this point. It is stuck at the following step:

![image](https://github.com/user-attachments/assets/7392623e-54c1-4936-ae1b-25f89ce610d6)

Just above this in the terminal it keeps reproducing the same 5 lines as seen below:

![image](https://github.com/user-attachments/assets/155c26d7-dc5d-403a-9430-6a4dcdd03cef)

I am using the internet connection of the TU so that cannot be the problem. What could be the problem here?

**Comment by yilun-wu** [comment, marked as answer]
I would try to do a rebuild without the cache. In VSCode, open the command palette and select and execute "Rebuild Container Without Cache". If this problem persists, please come to us tomorrow during the session.

**Reply by WouterOffringa** [reply]
This solved the issue. Thank you!

**Reply by mgutierreztudelft** [reply]
I am having the same problem. I had it all installed and working since last week but now the dev container needed to be ran again and it is stuck at the same place... The rebuild command does not appear to me in the command palette, should I stop the rebuid for this to appear? Thanks

**Reply by mgutierreztudelft** [reply]
OK so I stopped the process and still I cannot find the option of rebuild container without cache.

**Reply by yilun-wu** [reply]
see the answer above

> In VSCode, open the command palette and select and execute "Rebuild Container Without Cache".

**Reply by mgutierreztudelft** [reply]
Yes, the thing is the option "Rebuild Container Without Cache" does not appear in the command palette. Where can I find it? Thanks

**Reply by yilun-wu** [reply]
If it does not appear, perhaps you have not connected VSCode to WSL? Can you check what appears at the bottom left corner of VSCode, it should say WSL.

**Reply by mgutierreztudelft** [reply]
I do have it connected. Yesterday I was trying it with my room's wifi and perhaps the problem was that. I just tried it with TU wifi and it ran perfectly. Thanks for your help
